OJS.Workers.SubmissionProcessors/Workers/LocalSubmissionWorker.cs

```diff
@@ -8,8 +8,9 @@ public class LocalSubmissionWorker
     {
         private readonly int portNumber;
```

Comment:
This is not set anywhere, it should be removed

Reply:
It was removed